hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -699,7 +699,7 @@ public static boolean deleteDir(
             pairOfSubPathAndConf -> deleteSubPath(
                 pairOfSubPathAndConf.getKey(), pairOfSubPathAndConf.getValue(), true)
         );
-        boolean result = fs.delete(dirPath, false);
+        boolean result = fs.delete(dirPath, true);
```

Comment:
I'm confused, this deletion happens recursely from sub-dir to parent dir, why we need the `true` flag then?

Reply:
emm，in some extreme case sub-dir contains dir，cause not delete completely. then change it to true，all files can be delete rightly @danny0405